google/gax/__init__.py

```diff
@@ -31,11 +31,18 @@
 
 from __future__ import absolute_import
 import collections
+import logging
+import time
+import threading
+from concurrent import futures
+from .errors import GaxError
```

Comment:
Good catch. @eoogbe our Python style is to always use absolute imports. See https://google.github.io/styleguide/pyguide.html#Imports. I'm sure there's a PEP that says something similar.